components/gitpod-db/src/typeorm/entity/db-workspace-instance-usage.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { Entity, Column, PrimaryColumn, Index } from "typeorm";
+import { TypeORM } from "../typeorm";
+import { Transformer } from "../transformer";
+
+@Entity()
+export class DBWorkspaceInstanceUsage {
+    @PrimaryColumn(TypeORM.UUID_COLUMN_TYPE)
+    workspaceId: string;
+
+    @Column("varchar")
+    @Index()
+    attributionId: string;
+
+    @Column({
+        type: "timestamp",
+        precision: 6,
+        default: () => "CURRENT_TIMESTAMP(6)",
+        transformer: Transformer.MAP_ISO_STRING_TO_TIMESTAMP_DROP,
+    })
+    @Index()
+    startedAt: string;
+
+    @Column({
+        type: "timestamp",
+        precision: 6,
+        default: () => "CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6)",
```

Comment:
I'm not sure about the defaults here, I think we want this to be required but not to default to anything